bokehjs/src/lib/base.ts

```diff
@@ -88,3 +88,5 @@ export class ModelResolver {
 // TODO: this doesn't belong here, but it's easier this way for backwards compatibility
 import * as AllModels from "./models"
```

Comment:
Maybe rename this from `AllModels` to something more descriptive, since there are now also other `DOMModels`

Reply:
I will be replacing this with a more robust mechanism soon, where models will register themselves. This split between multiple groups of models is due to the current approach not allowing models with the same name (e.g. `widgets.Div` and `dom.Div`).